.github/workflows/add-label.yml

```
@@ -0,0 +1,33 @@
+
+jobs:
+  add-label:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+
+      - name: Determine label based on target branch
+        id: determine-label
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:3:23: Double quote to prevent globbing and word splitting

(shellcheck)